src/main/java/com/sarality/list/impl/CommonListItemViewHolder.java

```diff
@@ -12,4 +13,5 @@
   TextView titleTextView;
   TextView line2TextView;
   TextView line3TextView;
+  CheckBox secondaryActionCheckbox;
```

Comment:
Like we talked about, let's create a MessageListItemRenderer that takes care of this. We may need to rework the ViewHolder to include an ImageButton as well as text for Timestamp which is only used by the MessageList.

There are so lists where message appears at the bottom while for others it appears at top right. This can be supported by having multiple layouts but the rest of the code remains the same.